src/Components/Samples/BlazorUnitedApp/BlazorUnitedApp.csproj

```diff
@@ -4,12 +4,21 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <IsShippingPackage>false</IsShippingPackage>
     <Nullable>enable</Nullable>
+
+    <InterceptorsNamespaces>$(InterceptorsNamespaces);Microsoft.AspNetCore.Http.Validation.Generated</InterceptorsNamespaces>
```

Comment:
Hopefully, you won't need this soon once we take a new SDK update and absorb https://github.com/dotnet/sdk/pull/48891.